WordPress/src/main/java/org/wordpress/android/widgets/WPScrollView.java

```diff
@@ -35,37 +44,8 @@ public void setScrollDirectionListener(ScrollDirectionListener listener) {
     @Override
     public boolean onTouchEvent(MotionEvent event) {
         if (mScrollDirectionListener != null) {
```

Comment:
Yes, that's correct, but I admit it looks a bit weird. As you mention, `mDetector` is never null - it's just that we don't want to invoke it unless there's